### PR TITLE
add documentation for migrate tool

### PR DESCRIPTION
**Pull request by floren**

### CODE REVIEW

File: migrate/migrate.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+# Migration Tool
+Gravwell provides an interactive tool for migrating text files and Splunk data into Gravwell. This document describes how to install, configure, and use it.
+
+## Installation
+
+As the migration tool is still in early development, you’ll need to compile it yourself (or contact Gravwell for a binary file). Make sure you have [Go](https://golang.org) installed, then check out the repository and build the tool:
```
Comment (traetox) [inline migrate/migrate.md:6]: Its released, not early development.  Just show how to build it, we probably want to talk about building a "tools" package that delivers these into `/usr/local/sbin/gravwell_****`

File: migrate/migrate.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+After you have set the tag for the desired index + sourcetype pairs, you can select “Write Config” to write out a file in the `migrate.conf.d` directory which will store the mappings permanently.
+
+#### Starting Migrations
+
+Having defined mappings from Splunk index+sourcetype to Gravwell tag, you may now launch migration jobs. From the server menu, select “Start Migrations”. A menu will appear showing the index+sourcetype → tag mappings you defined earlier. Selecting one of these mappings will start a migration job, as seen in Figure @fig:migratejobs.
+
+![Migration jobs](migratejobs.png)
+
+You can launch multiple migrations at once. Note that Splunk migrations may take a while; if you exit the migrate tool while a Splunk migration is running, the job will be halted as soon as possible and the most recent timestamp will be stored to resume later–we make every effort to avoid data duplication\!
```
Comment (traetox) [inline migrate/migrate.md:127]: call out *HOW* to exit here?

**Review by traetox — CHANGES_REQUESTED** [review]
call out how to quit and remove the hesitation around code being ready.  It is ready.

**Comment by floren** [comment]
ptal

**Review by traetox — APPROVED** [review]
LGTM

**Pull request merged**